virtual_rainforest/core/__init__.py

```diff
@@ -0,0 +1,67 @@
+from virtual_rainforest.core.config import register_schema
+
+
+# This is definitely a bit questionable, its basically here to make sure that the plants
+# module is imported somewhere when tests are run, this ensures that the plant schema is
+# added to testing. I don't think this is the best or right way to do something like
+# this, but couldn't come up with a better way to do it
+# TODO - Think of a better solution for this
```

Comment:
One design consideration here might be conditional loading of schemas. As `vr` gets more complex, we could end up with more and more schema, and loading and validating them on request via the user configuration rather than hard coding them all  via `__init__.py` might be an advantage. 

Another thing is that we probably want that system in `__init__.py` to automatically discover schemas, rather than extensions having to manually add schema loading for `my_new_plant_module`.

But - that is all longer term.

Reply:
The test functions also do not use `schema` directly, they just need the schema registry to have be populated with the appropriate schema, so that the functions being tested can read the schema from there. So run into the same `flake8` pre-commit problem if I import there

Reply:
That particular error `F401` is sensible to ignore as the import is for a very good reason. So in those cases you just add `noqa` at the end like:
```python
from virtual_rainforest.plants import schema  # noqa
```